### ISSUE TITLE
Delegates? Protocols? or Reference assigned to a variable?

### ISSUE THREAD

**Issue by soulah** [xmartlabs/Eureka#1072]
Environment: Eureka 3.0.0, Xcode 8.3.2 and iOS 10.3.

I'm currently using class MyViewController: **MenuItemContentViewController** {}
(https://github.com/handsomecode/InteractiveSideMenu)

I receiving an EXC_BAD_ACCESS when doing the following:

```swift
class MyViewController: MenuItemContentViewController {
    @IBOutlet weak var formView: FormViewController!
    
    @IBAction func didOpenMenu(_ sender: UIButton) {
        showMenu()
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        formView.form +++ Section("Section1")
            <<< TextRow(){ row in
                row.title = "Text Row"
                row.placeholder = "Enter text here"
            }
            <<< PhoneRow(){
                $0.title = "Phone Row"
                $0.placeholder = "And numbers here"
            }
            +++ Section("Section2")
            <<< DateRow(){
                $0.title = "Date Row"
                $0.value = Date(timeIntervalSinceReferenceDate: 0)
        }
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }
}
```
Thanks in advance

Edit: Sorry, the title is no longer in regards to my original question.

**Comment by santiagofm** [comment]
Hey @soulah ! I think you want `MyViewController` to be both a `MenuItemContentViewController` and `FormViewController`, right ?
That way won't work, but maybe you can try adding the `FormViewController` as a `childViewController` and it's `view` as subview of `MyViewController`'s `view` matching it's bounds.
Let me know if that worked for you, cheers!

**Comment by soulah** [comment]
@santiagofm you sir are amazing. I actually tried to do that before but I used the wrong UI mistakingly. Don't laugh but I was just using view - haha!

For future people who have this problem, this is the outcome:
![18838778_10158826863580385_205998864_o](https://cloud.githubusercontent.com/assets/1690936/26657072/eaaf500c-462f-11e7-97ce-591971e6e83c.jpg)
![screen shot 2017-05-31 at 6 33 35 pm](https://cloud.githubusercontent.com/assets/1690936/26657079/f3edd300-462f-11e7-9d59-30620108f858.png)

Use a Container View inside your first controller.
Define the custom class on the newly generated view controller on main storyboard.

Thank you very much @santiagofm!
